property.go

```diff
@@ -13,12 +13,20 @@ type Property struct {
 
 type property struct {
 	label string
+	info  bool
 }
 
 // RegisterProperty registers a new property key.
 // It is used both to add a dynamic property to an error instance, and to extract property value back from error.
 func RegisterProperty(label string) Property {
-	return newProperty(label)
+	return newProperty(label, false)
+}
+
+// RegisterInfoProperty registers a new property key for informational value.
+// It is used both to add a dynamic property to an error instance, and to extract property value back from error.
+// Informational property will be included in Error() message.
+func RegisterInfoProperty(label string) Property {
```

Comment:
I'm not entirely happy about the name. Info gives a hint about logging (as in `log.Info`), but otherwise seems clumsy. 
`RegisterPrintableProperty`?
`RegisterMessageProperty`?